Sources/Control/Playback.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Playback.swift
+//  ControlKit
+//
+
+import AVFAudio
+import MediaPlayer
+
+public extension Control {
+    
+    enum Playback {
+        
+        /// Alias for `AVAudioSession.secondaryAudioShouldBeSilencedHint`.
+        public static var isAudioPlaying: Bool { avAudioSession.secondaryAudioShouldBeSilencedHint }
+        
+        private static let avAudioSession = AVAudioSession.sharedInstance()
+        
+        /// Toggles system media playback by activating or disactivating the shared `AVAudioSession`.
+        ///
+        /// Playback that has been paused by this function can normally be resumed if the app playing the content has not been terminated.
+        public static func togglePlayPause() {
+            try? avAudioSession.setActive(
+                isAudioPlaying,
+                options: .notifyOthersOnDeactivation
+            )
```

Comment:
Would `AVAudioSession` log an error/warning if `setActive` throws? 👀

What are your thoughts on the following given Playback has a logger?

`private static let log = Logger(subsystem: "com.ControlKit.Control", category: "Playback")`

```suggestion
            do {
                try avAudioSession.setActive(
                    isAudioPlaying,
                    options: .notifyOthersOnDeactivation
                )
            } catch {
                Self.log.warning(error.localizedDescription)
            }
```

Reply:
Yes, good call, might as well! 🕺